automation/roles/vip-manager/templates/vip-manager.yml.j2

```diff
@@ -64,16 +64,16 @@ etcd-user: {{ patroni_etcd_username | default("") }}
 {% if patroni_etcd_password | default("") | length > 0 %}
 etcd-password: {{ patroni_etcd_password | default("") }}
 {% endif %}
-{% if patroni_etcd_ca_file | default("") | length > 0 %}
+{% if tls_cert_generate | bool %}
 # when etcd-ca-file is specified, TLS connections to the etcd endpoints will be used.
-etcd-ca-file: {{ patroni_etcd_ca_file | default("") }}
+etcd-ca-file: {{ tls_ca_cert_path | default('/etc/tls/ca.crt') }}
 {% endif %}
-{% if patroni_etcd_cert_file | default("") | length > 0 %}
+{% if tls_cert_generate | bool %}
 # when etcd-cert-file and etcd-key-file are specified, we will authenticate at the etcd endpoints using this certificate and key.
-etcd-cert-file: {{ patroni_etcd_cert_file | default("") }}
+etcd-cert-file: {{ tls_cert_path | default('/etc/tls/server.crt') }}
 {% endif %}
-{% if patroni_etcd_key_file | default("") | length > 0 %}
-etcd-key-file: {{ patroni_etcd_key_file | default("") }}
+{% if tls_cert_generate | bool %}
+etcd-key-file: {{ tls_privatekey_path | default('/etc/tls/server.key') }}
```

Comment:
I think one condition `{% if tls_cert_generate | bool %}` is enough for all TLS-related options.